gatling-core/src/main/scala/io/gatling/core/session/el/ElCompiler.scala

```diff
@@ -284,6 +284,24 @@ final case class RandomLongRange(min: Long, max: Long) extends ElPart[Long] {
   def apply(session: Session): Validation[Long] = (min + ThreadLocalRandom.current().nextLong(length)).success
 }
 
+final case class RandomDoubleRange(min: Double, max: Double) extends ElPart[Double] {
+  require(min < max, s"'max'($max) should be greater than 'min'($min)")
+  private val minInclusive = min
+  private val maxExclusive = max
+  def apply(session: Session): Validation[Double] =
+    ThreadLocalRandom.current().nextDouble(minInclusive, maxExclusive).success
+}
+
+final case class RandomDoubleRangeDigits(min: Double, max: Double, fractionalDigits: Int) extends ElPart[Double] {
+  require(min < max, s"'max'($max) should be greater than 'min'($min)")
+  require(fractionalDigits >= 0, s"fractionalDigits(${fractionalDigits}) must be positive")
+  private val minInclusive = min
+  private val maxExclusive = max
```

Comment:
What do you need those references for?

Reply:
Sorry about that, these are left overs, will remove these.